api.planx.uk/package.json

```diff
@@ -37,7 +37,7 @@
     "mime": "^3.0.0",
     "multer": "^1.4.5-lts.1",
     "nanoid": "^3.3.7",
-    "notifications-node-client": "^7.0.6",
+    "notifications-node-client": "^8.0.0",
```

Comment:
No changes that affect us with this major version bump https://github.com/alphagov/notifications-node-client/blob/main/CHANGELOG.md